### DISCUSSION TITLE
Xbox 360 Controller with Chatpad.

### DISCUSSION THREAD

**Discussion by EramarkMedia** [AntiMicroX/antimicrox#333] [General]
Hi everyone.

I have been developing a piece of software, that i would like to use with some agricultural / forestry applications in my tractor / other various machines.

Since it would not be very nice, to have a keyboard and mouse combo, rolling around in the cab, getting jammed under tools etc, i was thinking. That using a Xbox 360 controller, with Antimicro / Antimicrox, would be a fairly compact solution.

However, i would also like to use the chatpad, as a Keyboard, because well, it would be the ultimate compact combo.
I have seen some notes, about that chatpad support, should at least partially be there, or in the making, but so far, my tests with actually using it, have been unsuccessful.

I have not really been able to receive raw events from the chatpad either.

Could someone more familiar with antimicrox, spread some light on me, if the chatpad is even supported currently ?

Have a nice day :)

**Comment by pktiuk** [comment]
Have you tried to check this chatpad with other apps like `qjoypad` or `jstest-gtk`?

I think Xbox 360 chatpad is not fully supported by default Linux driver.  
You could try with other drivers like: https://gitlab.com/xboxdrv/xboxdrv
